ci/functional/test_main_node.sh

```diff
@@ -10,5 +10,6 @@ export TEST_RPMS=true
 export REMOTE_ACCT=jenkins
 export WITH_VALGRIND="$WITH_VALGRIND"
 export STAGE_NAME="$STAGE_NAME"
+export COVFN_DISABLED="$COVFN_DISABLED"
```

Comment:
Let's add a default here incase it's not passed:
```suggestion
export COVFN_DISABLED="${COVFN_DISABLED:-true}"
```